.gitignore

```diff
@@ -17,6 +17,7 @@ client/build
 .env.test.local
 .env.production.local
 .env
+.rej
```

Comment:
what are these files?

Reply:
1. We might create .env files for environment variables for production, testing environments. These files should not be committed into code.
2. .rej are reject files when we apply patch.